speech-synthesis/selecting-prompts.md

```diff
@@ -0,0 +1,75 @@
+# Selecting good prompts
+By now you have a dataset(s) with text from which you want to select the best sentences(prompts) to build your speech corpus. This section takes you through how to obtain the best utterances from your dataset(s). Most of the information here is obtained from [the corpus development chapter](http://festvox.org/bsv/c2176.html) where you can find detailed explanations.
```

Comment:
sentences(prompts)
-> sentences (prompts)

best utterances
-> best prompts

the corpus development chapter
-> the corpus development chapter *of the festvox toolkit*